packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -752,7 +755,9 @@ class FlutterPlugin implements Plugin<Project> {
      * from the command line.
      *
      * The AGP team said that this issue is fixed in Gradle 7.0, which isn't released at the
-     * time of adding this code. Once released, this can be removed.
+     * time of adding this code. Once released, this can be removed. However, after updating to
+     * AGP/Gradle 7.2.0/7.5, removing this hack still causes build failures. Futher
+     * investigation necessary to remote this.
```

Comment:
Is there an issue we can reference here in a TODO?

Reply:
Let me file something and add a bit more detail.